src/statmodels.jl

```diff
@@ -79,6 +101,37 @@ when applicable the intercept and the distribution's dispersion parameter.
 """
 dof(obj::StatisticalModel) = error("dof is not defined for $(typeof(obj)).")
 
+"""
+    mss(obj::StatisticalModel)
+
+Return the model sum of squares.
+"""
+function mss(obj::StatisticalModel)
+    μ = meanresponse(obj)
```

Comment:
Should be `m`. (We really need to add tests in StatsBase for these methods at some point...)

Also, use the same approach for other `sum` calls.